src/main/java/com/wheretoeat/WhereToEatController.java

```diff
@@ -2,20 +2,70 @@
 
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+import com.wheretoeat.dto.Option;
+
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 import org.springframework.boot.SpringApplication;
 
 @Controller
 @SpringBootApplication
 public class WhereToEatController {
 
-	@RequestMapping("/start")
-	public String start() {
-		return "start";
-	}
-	public static void main(String... args) throws Exception {
-        SpringApplication.run(WheretoeatApplication.class, args);
-    }
-
-	
+	@RequestMapping(value="/start", method=RequestMethod.GET)
+	public ModelAndView start() {
+		List<Option> foodTypeOptions = new ArrayList<Option>();		

```

Comment:
- Dynamic dropdown options. It is recommended that you get dropdown options from DB rather than hardcode. So that you don't have to redeploy the app every time you change the name of the option
- Please ignore or delete my comment code. I forgot to remove that